src/app/[locale]/(routes)/collection/page.tsx

```diff
@@ -1,14 +1,78 @@
+"use client";
 import Cardcontainer from "@/app/component/common/Cards/cardcontainer";
-import { FC } from "react";
-
-interface pageProps {
-    
-}
- 
-const page: FC<pageProps> = () => {
-    return ( <>
-    <Cardcontainer/>
-    </> );
-}
- 
-export default page;
\ No newline at end of file
+import { FC, useState } from "react";
+import { useAccount, useBlockNumber } from "wagmi";
+import Balance from "@/app/component/functions/BalanceFinder";
+import { LayoutRouterContext } from "next/dist/shared/lib/app-router-context.shared-runtime";
+import { parse } from "path";
+
+interface pageProps {}
+
+const Page: FC<pageProps> = () => {
```

Comment:
Im not sure what is doing this component. 

Maybe change also the name `Page`

Reply:
done